pkg/ride/compiler/ast_parser_test.go

```diff
@@ -868,23 +869,23 @@ func cursed() = [][0]
 	}
 }
 
-////go:embed test_scripts
-//var embedScripts embed.FS
-
-//	func TestBigScripts(t *testing.T) {
-//		cli, err := client.NewClient(client.Options{
-//			BaseUrl: "https://nodes.wavesnodes.com",
-//			Client:  &http.Client{Timeout: 10 * time.Second},
-//		})
-//		require.NoError(t, err)
-//		files, err := embedScripts.ReadDir("test_scripts")
-//		require.NoError(t, err)
-//		for _, file := range files {
-//			t.Logf("Test %s", file.Name())
-//			code, err := embedScripts.ReadFile("test_scripts/" + file.Name())
-//			require.NoError(t, err)
-//			res, _, err := cli.Utils.ScriptCompile(context.Background(), string(code))
-//			require.NoError(t, err)
-//			compareScriptsOrError(t, string(code), false, strings.TrimPrefix(res.Script, "base64:"))
-//		}
-//	}
+//go:embed test_scripts
+var embedScripts embed.FS
+
```

Comment:
Shouldn't we execute tests that require communication with an external resource (node) only when a special option is provided?"
```suggestion
var integration bool

func init() {
	flag.BoolVar(&integration, "integration", false, "Execute integration tests. False by default.")
}

func TestBigScripts(t *testing.T) {
	if !integration {
		t.Skip("Skipping integration test")
	}

```
To run the test the `-integration` flag should be provided:
```bash
go test -integration ./ast_parser_test.go
```